examples/solids/README.rst

```diff
@@ -0,0 +1,178 @@
+libCEED: Solid Mechanics Example
+--------------------------------
+
+This page provides a description of the solid mechanics example for the
+libCEED library, based on PETSc.
+
+This code solves the steady-state static momentum balance equations using unstructured high-order finite/spectral element spatial discretizations.
+In this mini-app, we consider three formulations used in solid mechanics applications: linear elasticity, Neo-Hookean hyperelasticity at small strain, and Neo-Hookean hyperelasticity at finite strain.
+
+Build by using::
+
+   make
+
+and run with::
+
+   ./elasticity -mesh [.exo file] -degree [degree] -nu [nu] -E [E] [boundary options] -problem [problem type] -forcing [forcing] -ceed [ceed]
+
+Runtime options
+---------------
+
+.. inclusion-marker-do-not-remove
+
+The elasticity min-app is controlled via command-line options, the following of which are mandatory.
+
+.. list-table:: Mandatory Runtime Options
+   :header-rows: 1
+
+   * - Option
+     - Description
+
+   * - :code:`-mesh [filename]`
+     - Path to mesh file in any format supported by PETSc.
+
+   * - :code:`-degree [int]`
+     - Polynomial degree of the finite element basis
+
+   * - :code:`-E [real]`
+     - `Young's modulus <https://en.wikipedia.org/wiki/Young%27s_modulus>`_, :math:`E > 0`
+
+   * - :code:`-nu [real]`
+     - `Poisson's ratio <https://en.wikipedia.org/wiki/Poisson%27s_ratio>`_, :math:`\nu < 0.5`
+
+   * - :code:`-bc_zero [int list]`
+     - List of faces sets on which to enforce zero displacement
+
+   * - :code:`-bc_clamp [int list]`
+     - List of face sets on which to displace by :code:`-bc_clamp_max` in the :math:`y` direction
+
+(One can set only one of :code:`-bc_zero` or :code:`-bc_clamp`, but the result will likely not be interesting.)
+The following is an example of a minimal set of command line options::
+
+   ./elasticity -mesh [.exo file] -degree 4 -E 1e6 -nu 0.3 -bc_zero 999 -bc_clamp 998
```

Comment:
Where can a reference to these numbers be found? How do I know what `999` or `998` mean?

Reply:
in the sample meshes file. The numbers depend of the mesh file

Reply:
Yes, but it should be documented and explained. Perhaps we should add this [diagram](https://github.com/jeremylt/ceedSampleMeshes/blob/master/cylinderDiagram.png) to this repo docs as well. Otherwise those are numbers that for someone who wants to test the code are meaningless.

Reply:
I don't think that diagram makes any sense here. This code is not specific to that mesh.

Personally, I think its fine. These are just sample face numbers. If you want, we can use
```
./elasticity -mesh [mesh] -degree [int] -E [real] -nu [real] -bc_zero [int list] -bc_clamp [int list]
```
to make it totally obvious that the user has to specify everything for the exact problem they want to solve. Or we can remove this sample to avoid the controversy altogether.

Reply:
FWIW, you can include it directly from Jeremy's repository.
```
.. image:: https://github.com/jeremylt/ceedSampleMeshes/raw/master/cylinderDiagram.png
```
Someone not familiar with mesh formats may be confused about what is meant. It would be nice to link an image marking the surfaces and a low-res visualization of a solution.